addons/survey/models/survey_survey.py

```diff
@@ -175,6 +174,8 @@ def default_get(self, fields_list):
     session_show_leaderboard = fields.Boolean("Show Session Leaderboard", compute='_compute_session_show_leaderboard',
         help="Whether or not we want to show the attendees leaderboard for this survey.")
     session_speed_rating = fields.Boolean("Reward quick answers", help="Attendees get more points if they answer quickly")
```

Comment:
Hello @flch-odoo , just a few questions / remarks after rereading the diff.

1. This option is a bit confusing. What it does is setting a time limit to all non-customized questions, right? However, if not checked, the questions with is_time_limited and a time_limit will still be rewarded 
2. The question form has a 'Live Session' Section no matter the type of survey it is on. But the survey will not have a 'Live session' section displayed for all types. What do you think, should we leave it as is or align both?
3. the help field of session_speed_rating_time_limit is not strictly correct as we will lose some point with time elapsing and not receive some additional ones, if I get it right. Not sure how we could put it better though.
4. Probably to discuss with Francois during testing, but while I'm here I'm wondering if setting is_time_customized to False is relevant when a previously customized has the same duration as survey again. But no better solution comes to mind without making the field stuck with False value and survey less usable, so probably a thought going straight to oblivion, nvm... :man_shrugging:

Cheers!

Reply:
On my side, the other things that come to mind are functional, so I would say we can move to testing.

Reply:
Thank you for the remarks, @Odoonan . As you know I've brought this to the attention of François when appropriate and here's what came out
For point 1, this was a mistake on my part, which should have now corrected. 
For point 2, this feature is currently only supported for live sessions and we'll keep it this way.
For point 3, wording will be reworked in testing BE.
For point 4, ... oh, there's no point 4^^

Reply:
Hello @flch-odoo , thank you for the reply. Actually for point 2, my point was only concerning the UI, not the feature range of application.

I've just noticed that the survey form has a conditional 'Live Session' Section http://tinyurl.com/2cop6pc8
But the section on the question form is always there. I was wondering if we wanted to align both forms or keep it as is :) http://tinyurl.com/2xo2quak

No big deal. I'd say we can move it to testing. I do not know if TDE had time to have another look but this must still goes through review so I say let it move forward.

Reply:
Mmh,

yes, help may be good enough as we would need to add related fields on survey.question to check for session_speed_rating and / or survey_type to make a conditional display.

Let us keep it simple for now, can be added downstream if deemed necessary. I'll green once the runbot is done :)